ui/pages/confirmations/components/send/recipient/recipient.tsx

```diff
@@ -193,9 +114,13 @@ export const Recipient = ({
         </HelpText>
       )}
       {to === toAddressValidated && recipientResolvedLookup && (
-        <HelpText severity={HelpTextSeverity.Info} marginTop={1}>
-          {t('resolvedLookup', [recipientResolvedLookup])}
-        </HelpText>
+        <Text
+          color={TextColor.textAlternative}
+          marginTop={1}
+          variant={TextVariant.bodyXs}
+        >
+          {`${t('resolutionProtocol')} ${resolutionProtocol ?? ''}`}
```

Comment:
Ideally we shouldn't modify displayed strings with hardcoded spaces like this, because some languages change the order of the phrase. We can pass resolutionProtocol as an argument to the translation function like it was previously done for recipientResolvedLookup, and use $1 in the translation string.

Reply:
Good point, I updated the PR